src/main/java/lotto/domain/Lotto.java

```diff
@@ -34,6 +40,24 @@ public int numberOfSame(Lotto lotto) {
                 .count();
     }
 
+    public boolean isBonusNumber(BonusNumber bonusNumber) {
+        return this.lottoNumbers.stream().anyMatch(number -> bonusNumber.isSameNumber(number));
```

Comment:
```suggestion
        return this.lottoNumbers.stream().anyMatch(bonusNumber::isSameNumber);
```

메서드 레퍼런스 형태로도 적용해 볼 수 있겠네요 😄